masterlist.yaml

```
@@ -383,8 +383,8 @@ common:
     <<: *patch3rdParty
     subs:
       - '[Spell Research v2.2+](https://www.nexusmods.com/skyrimspecialedition/mods/20983/)'
-      - '[Spell Research - Patch Compendium](https://www.nexusmods.com/skyrimspecialedition/mods/28816/)'
-    condition: 'active("SpellResearch.esp") and not active("Spell Research - Patch Compendium.esp") and not checksum("SpellResearch.esp", 755423D7) and not checksum("SpellResearch.esp", 63661836) and not checksum("SpellResearch.esp", 860EFC3D)'
+      - '[Spell Research - Patch Compendium Redux](https://www.nexusmods.com/skyrimspecialedition/mods/61177)'
```

Add trailing slash to link.
```suggestion
      - '[Spell Research - Patch Compendium Redux](https://www.nexusmods.com/skyrimspecialedition/mods/61177/)'
```